content/pages/404.md

```diff
@@ -22,7 +22,7 @@ permalink: false
           <p>
             Try the search box or one of the common questions below.
           </p>
-          <div class="call-out">
+          <div class="feature">
```

Comment:
Yeah ... that's only part of the problem, it seems. I think I got a little aggressive with some refactoring to remove CSS